README.md

```diff
@@ -106,6 +106,21 @@ end
 
 Operator match data includes `match['operator']` and `match['expression']`. The `bot` match always checks against the `SlackRubyBot::Config.user` setting.
 
+You can reply to a message in a thread (or keep sending to the current thread) :
+
+```ruby
+command('reply in thread') do |client, data, match|
```

Comment:
Can we remove the parentheses for consistency with the rest of the documentation